Gameplay.java

```diff
@@ -0,0 +1,226 @@
+package brickBreaker;
+
+import java.awt.Color;
+import java.awt.Font;
+import java.awt.Graphics;
+import java.awt.Graphics2D;
+import java.awt.Rectangle;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.event.KeyEvent;
+import java.awt.event.KeyListener;
+import javax.swing.Timer;
+
+import javax.swing.JPanel;
+
+public class Gameplay extends JPanel implements KeyListener, ActionListener {
+    private boolean play = false;
+    private int score = 0;
+     
+    private int totalBricks = 21;
+    
+    private Timer timer;
+    private int delay = 8;
+    
+    private int playerX = 300;
+    
+    private int ballposX =120;
+    private int ballposY = 350;
+    private int ballXdir = -1;
+    private int ballYdir = -2;
+    
+    private MapGenerator map;
+    
+    
+    public Gameplay()
+    {
+    	map = new MapGenerator(5,8);
+    	addKeyListener(this);
+    	setFocusable(true);
+    	setFocusTraversalKeysEnabled(false);
+    	timer = new Timer(delay, this);
+      	timer.start();
```

Comment:
Is it possible to use a formatter to automatically align your code, provide proper and consistent spaces etc?